test/utils/redux.js

```diff
@@ -12,10 +13,13 @@ function restoreAppStore() {
 }
 
 export function createMockedStore(sinon, mockedState = {}) {
-    const store = createStore(() => mockedState);
+    const middlewares = [thunkMiddleware];
+    const mockStore = configureStore(middlewares);
+    const store = mockStore(mockedState);
 
     sinon.spy(store, 'dispatch');
     store.restore = restoreAppStore;
+    store.subscribe = () => Function();
```

Comment:
@lemieux See above. Invoking `Function` like this `Function()` will just return an empty function.

I'll change it so `Function()` is not used anymore